src/main/java/duke/error/IncorrectFormat.java

```diff
@@ -0,0 +1,7 @@
+package duke.error;
+
+public class IncorrectFormat  extends Exception{
```

Comment:
Did you accidentally add an extra space after before `extends`?
In addition, `IncorrectFormat` seem not to be the conventional way of naming an exception. Perhaps `IncorrectFormatException`? Or maybe even more specific.
Furthermore, maybe add a JavaDoc for the exception would be better because `IncorrectFormat` is still vague and ambiguous